tests/visualization_tests/matplotlib_tests/test_pareto_front.py

```diff
@@ -491,3 +491,16 @@ def test_plot_pareto_front_using_axis_order_and_targets() -> None:
             axis_order=[0, 1, 2],
             targets=lambda t: (t.values[0], t.values[1], t.values[2]),
         )
+
+
+def test_constraints_func_experimental_warning() -> None:
+    study = optuna.create_study(directions=["minimize", "minimize"])
+
+    def constraints_func(t: FrozenTrial) -> Sequence[float]:
+        return [1.0] if t.params["x"] == 1 and t.params["y"] == 0 else [-1.0]
+
+    with pytest.warns(optuna.exceptions.ExperimentalWarning):
+        plot_pareto_front(
+            study=study,
+            constraints_func=constraints_func,
+        )
```

Comment:
The `constraints_func` seems to be a bit complicated. It refers to `t.params["x"]` and `t.params["y"]` although the study is empty.

Developers might be able to focus on the experimental warnings if we use simple `constraints_func`. The following is just an example, but can we remove the `t.params` in `constraints_func`?

```
    with pytest.warns(optuna.exceptions.ExperimentalWarning):
        plot_pareto_front(study=study, constraints_func=lambda _: [1.0])
```

Reply:
Thanks, I applied your suggestion.